### PR TITLE
Comment popup

### PR DESCRIPTION
**Pull request by ridwanediallo**
Hello @brahimdidi, I come to finish the first part of my work. Would you want to review it for me? 
Thank you.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,11 +1,68 @@
...
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" id="exampleModalLabel">Modal title</h5>
+          <img src="" alt="">
+          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+        </div>
+        <div class="modal-body">
+          <div class="card mb-3">
+            <img src="./garden-g3d47ba2a6_1920.jpg" class="card-img-top" alt="...">
```
Comment (brahimdidi) [inline src/index.html]: This image should have an alt for cleaner code, please assess that.

File: src/index.html [hunk 2]
```diff
@@ -1,11 +1,68 @@
...
+            <div class="card-body">
+              <div class="row">
+                <div class="col-sm-4">
+                  <h5 class="card-title">Add comment</h5>
+                  <form class="mb-3">
+                    <input type="text" class="form-control mb-3" id="your-name" placeholder="Your name">
```
Comment (brahimdidi) [inline src/index.html]: This input should have a label for cleaner code

**Review by brahimdidi — APPROVED** [review]
STATUS: APPROVED ✔️ ✔️
Hi [@ridwanediallo ](https://github.com/ridwanediallo),

Your project is complete! However, I would like to suggest some changes :shipit:
Congratulations! 🎉



I would suggest using fewer divs and more tags like p, main, article...
Remember You will have to use javascript to implement the popup

**Comment by ridwanediallo** [comment]
@brahimdidi thank you for the review.

**Pull request merged**